pkg/api.go

```diff
@@ -0,0 +1,85 @@
+package sfu
+
+import (
+	"encoding/json"
+
+	"github.com/pion/ion-sfu/pkg/log"
+	"github.com/pion/webrtc/v3"
+)
+
+// All available data channels commands for SFU, may reflect only changes on caller,
+// this is not intended to force remove,mute, etc. to other publisher tracks.
+// Data channel commands are received in ion-sfu data channel. So it must be considered
+// a reserved name
+const (
+	// Reserved data channel streamID for sfu commands
+	channelLabel = "ion-sfu"
+
+	// Mute command
+	// Direction: Client -> SFU -> Client
+	// Client: Send intent to mute the stream
+	// Server: Send confirmation that stream is muted
+	muteCommand = "mute"
+
+	// Unmute command
+	// Direction: Client -> SFU -> Client
+	// Client: Send intent to unmute the stream
+	// Server: Send confirmation that stream is un-muted
+	unmuteCommand = "unmute"
+
+	// Best Quality Command
+	// Direction: Client -> SFU -> Client
+	// Client: Send intent to get best quality of the stream
+	// Server: Send confirmation that stream is in best quality,
+	// send an error if stream does not support simulcast/SVC
+	// NOTE: client may not get best quality is bw is a limitation.
+	forceBestQuality = "bestQuality"
```

Comment:
i'm not sure if we want to allow the client to "force". i feel like they should request but can be overridden by the sfu's congestion control. for example, if a speaker is active and large screen, i would like their high res but wouldn't want to force it if it degrades the stream. i feel like the client should always be a soft preference. but maybe i am missing a use case

Reply:
oh i just saw your comment `NOTE: client may not get best quality is bw is a limitation`. maybe it could be `preferBestQuality`